pilot/cmd/pilot-agent/status/server.go

```diff
@@ -694,7 +692,7 @@ func (s *Server) handleQuit(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	_, _ = w.Write([]byte("OK"))
 	log.Infof("handling %s, notifying pilot-agent to exit", quitPath)
-	s.shutdown()
+	s.shutdown(fmt.Errorf("%v called", quitPath))
```

Comment:
nit: with this, we probably do not need the `Info` log telling us `/quitquitquit` is being handled anymore.

Reply:
I think its still useful since /quitquitquit call is a kind of Big Deal, and its not guaranteed we will even call the context.Cause -- currently, its only triggered when k8s probes us which only happens like every 15s IIRc